docs/proving-authentication.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Proving Ownership of a Flow Account
+---
+
+## Proving Ownership of a Flow Account
+
+A common desire that application developers have is to be able to prove that a
+user controls an on chain account. Fortunately, FCL provides multiple ways to
+achieve this.
+
+During user authentication, some FCL compatible wallets will choose to support
+the FCL `account-proof` service. If a wallet chooses to support this service,
+they will include within it specific pieces of information that you can use to
+prove that a user controls an on chain account. The service, and its data within
+it, will be returned from the wallet during authentication, and made available
+for your use in `FCL Current User`. 
+
+If a user's wallet does not choose to support the `account-proof` service, you
+can still prove that a user controls an on chain account provided that the
+wallet does support the `user-signature` service. Since all wallets may
+optionally choose to support either of these services, your application should
+be resilient enough to handle cases where a user's wallet supports one, both or
+neither.
+
+The `account-proof` service, as the name implies, was designed specifically for
+proving authentication, while `user-signature` was designed for more generic use
+cases. We suggest that your application choose to use `account-proof` service
+first, if it's available, and then fall back to `user-signature` if needed.
+
+We'll walk through how you, an application developer, can use either service to
+prove a user's authentication.
+
+> Are you an FCL Wallet Developer? Check out the wallet provider specific docs
+[here](https://github.com/onflow/fcl-js/tree/master/packages/fcl/src/wallet-provider-spec/provable-authn.md)
+
+### Proving User Authentication using `account-proof`
+
+The suggested order of operations of how your application might use
+`account-proof` begins with:
+
+- Optional: Your application may choose to configure FCL with an application
+  specific Domain Tag. A Domain Tag helps scope the cryptographic proof
+  generated for the `account-proof` service to your application. Your
+  application can configure a Domain Tag like:
+
+```javascript
+import {config} from "@onflow/fcl"
+
+config({
+    "fcl.domainTag": "YourApplicationsDomainTag"
```

Comment:
is this a valid domain tag? can we add some details here on what makes a valid domain tag

Reply:
Added details on app domain tags => https://github.com/onflow/fcl-js/pull/793/files#diff-6ceb14c9b18fcb71d48ff6aa3f1a363c4fd1dd369a16bf69a1c8f7d2329103e8R43-R44